packages/template-retail-react-app/app/routes.jsx

```diff
@@ -114,7 +101,22 @@ export const routes = [
     },
     {
         path: '*',
-        component: PageNotFound
+        component: withRouter((props) => {
+            const {location} = props
+            const urlParams = new URLSearchParams(location.search)
+
+            useEffect(() => {
+                const newURL = new URL(window.location)
+                if (!urlParams.has('redirected')) {
+                    newURL.searchParams.append('redirected', '1')
+                    window.location.href = newURL
+                }
+            }, [window.location.href])
+            if (urlParams.has('redirected')) {
+                return <PageNotFound {...props} />
+            }
+            return null
+        })
```

Comment:
These changes for hybrid setup allow for easy deployments to storefronts. These changes are not to be merged in to develop.